File: clients/java/android/src/main/java/com/wordnik/client/model/Affiliate.java
package com.wordnik.client.model;

import com.fasterxml.jackson.annotation.JsonProperty;

import java.util.Date;
public class Affiliate {
  @JsonProperty("account")
  private Double account = null;
  @JsonProperty("currency")
  private String currency = null;
  @JsonProperty("prevPayout")
  private Double prevPayout = null;
  @JsonProperty("prevTurnover")
  private Double prevTurnover = null;
  @JsonProperty("prevComm")
  private Double prevComm = null;
  @JsonProperty("prevTimestamp")
  private Date prevTimestamp = null;
  @JsonProperty("execTurnover")
  private Double execTurnover = null;
  @JsonProperty("execComm")
  private Double execComm = null;
  @JsonProperty("totalReferrals")
  private Double totalReferrals = null;
  @JsonProperty("totalTurnover")
  private Double totalTurnover = null;
  @JsonProperty("totalComm")
  private Double totalComm = null;
  @JsonProperty("payoutPcnt")
  private Double payoutPcnt = null;
  @JsonProperty("pendingPayout")
  private Double pendingPayout = null;
  @JsonProperty("timestamp")
  private Date timestamp = null;
  public Double getAccount() {
    return account;
  }
  public void setAccount(Double account) {
    this.account = account;
  }

  public String getCurrency() {
    return currency;
  }
  public void setCurrency(String currency) {
    this.currency = currency;
  }

  public Double getPrevPayout() {
    return prevPayout;
  }
  public void setPrevPayout(Double prevPayout) {
    this.prevPayout = prevPayout;
  }

  public Double getPrevTurnover() {
    return prevTurnover;
  }
  public void setPrevTurnover(Double prevTurnover) {
    this.prevTurnover = prevTurnover;
  }

  public Double getPrevComm() {
    return prevComm;
  }
  public void setPrevComm(Double prevComm) {
    this.prevComm = prevComm;
  }

  public Date getPrevTimestamp() {
    return prevTimestamp;
  }
  public void setPrevTimestamp(Date prevTimestamp) {
    this.prevTimestamp = prevTimestamp;
  }

  public Double getExecTurnover() {
    return execTurnover;
  }
  public void setExecTurnover(Double execTurnover) {
    this.execTurnover = execTurnover;
  }

  public Double getExecComm() {
    return execComm;
  }
  public void setExecComm(Double execComm) {
    this.execComm = execComm;
  }

  public Double getTotalReferrals() {
    return totalReferrals;
  }
  public void setTotalReferrals(Double totalReferrals) {
    this.totalReferrals = totalReferrals;
  }

  public Double getTotalTurnover() {
    return totalTurnover;
  }
  public void setTotalTurnover(Double totalTurnover) {
    this.totalTurnover = totalTurnover;
  }

  public Double getTotalComm() {
    return totalComm;
  }
  public void setTotalComm(Double totalComm) {
    this.totalComm = totalComm;
  }

  public Double getPayoutPcnt() {
    return payoutPcnt;
  }
  public void setPayoutPcnt(Double payoutPcnt) {
    this.payoutPcnt = payoutPcnt;
  }

  public Double getPendingPayout() {
    return pendingPayout;
  }
  public void setPendingPayout(Double pendingPayout) {
    this.pendingPayout = pendingPayout;
  }

  public Date getTimestamp() {
    return timestamp;
  }
  public void setTimestamp(Date timestamp) {
    this.timestamp = timestamp;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class Affiliate {\n");
    sb.append("  account: ").append(account).append("\n");
    sb.append("  currency: ").append(currency).append("\n");
    sb.append("  prevPayout: ").append(prevPayout).append("\n");
    sb.append("  prevTurnover: ").append(prevTurnover).append("\n");
    sb.append("  prevComm: ").append(prevComm).append("\n");
    sb.append("  prevTimestamp: ").append(prevTimestamp).append("\n");
    sb.append("  execTurnover: ").append(execTurnover).append("\n");
    sb.append("  execComm: ").append(execComm).append("\n");
    sb.append("  totalReferrals: ").append(totalReferrals).append("\n");
    sb.append("  totalTurnover: ").append(totalTurnover).append("\n");
    sb.append("  totalComm: ").append(totalComm).append("\n");
    sb.append("  payoutPcnt: ").append(payoutPcnt).append("\n");
    sb.append("  pendingPayout: ").append(pendingPayout).append("\n");
    sb.append("  timestamp: ").append(timestamp).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: clients/csharp/src/Swagger/Client/Model/Affiliate.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Swagger.Client.Model {
  public class Affiliate {
    public double? account { get; set; }

    public string currency { get; set; }

    public double? prevPayout { get; set; }

    public double? prevTurnover { get; set; }

    public double? prevComm { get; set; }

    public DateTime? prevTimestamp { get; set; }

    public double? execTurnover { get; set; }

    public double? execComm { get; set; }

    public double? totalReferrals { get; set; }

    public double? totalTurnover { get; set; }

    public double? totalComm { get; set; }

    public double? payoutPcnt { get; set; }

    public double? pendingPayout { get; set; }

    public DateTime? timestamp { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class Affiliate {\n");
      sb.Append("  account: ").Append(account).Append("\n");
      sb.Append("  currency: ").Append(currency).Append("\n");
      sb.Append("  prevPayout: ").Append(prevPayout).Append("\n");
      sb.Append("  prevTurnover: ").Append(prevTurnover).Append("\n");
      sb.Append("  prevComm: ").Append(prevComm).Append("\n");
      sb.Append("  prevTimestamp: ").Append(prevTimestamp).Append("\n");
      sb.Append("  execTurnover: ").Append(execTurnover).Append("\n");
      sb.Append("  execComm: ").Append(execComm).Append("\n");
      sb.Append("  totalReferrals: ").Append(totalReferrals).Append("\n");
      sb.Append("  totalTurnover: ").Append(totalTurnover).Append("\n");
      sb.Append("  totalComm: ").Append(totalComm).Append("\n");
      sb.Append("  payoutPcnt: ").Append(payoutPcnt).Append("\n");
      sb.Append("  pendingPayout: ").Append(pendingPayout).Append("\n");
      sb.Append("  timestamp: ").Append(timestamp).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }
